x-pack/plugins/alerting/server/saved_objects/mappings.json

```diff
@@ -155,6 +155,10 @@
             }
           }
         }
+      },
+      "snoozeEndTime": {
+        "type": "date",
+        "format": "strict_date_time"
```

Comment:
I think we should use this instead of `strict_date_time_no_millis`. `Date.toISOString()` outputs with milliseconds by default, so this would be easier to work with.